### PR TITLE
Pat 1885 node update security dependencies

### PR DESCRIPTION
**Pull request by steverendell**

### CODE REVIEW

File: server/database/index.js [hunk 1]
```diff
@@ -8,7 +9,13 @@ const knex = require('knex')({
...
     password: config.db.password,
-    ssl: config.db.sslEnabled === 'true',
+    ssl:
+      config.db.sslEnabled === 'true'
+        ? {
+          ca: fs.readFileSync('root.cert'),
+          rejectUnauthorized: true,
+        }
+        : false,
   },
```
Comment (steverendell) [inline server/database/index.js:19]: slipped this in - will be needed when going to node 14

File: server/services/userAuthenticationService.js [hunk 2]
```diff
@@ -13,7 +13,7 @@ const getUserAuthenticationDetails = async (quantumId) => {
 const updateUserSessionExpiryAndLastLoginDateTime = async (quantumId, dateTime) => {
   db('UserAuthentication')
     .where({ QuantumId: quantumId.toLowerCase() })
-    .update({ LastLoginDateTime: new Date().toLocaleString(), SessionExpiryDateTime: dateTime })
+    .update({ LastLoginDateTime: new Date(), SessionExpiryDateTime: dateTime })
```
Comment (steverendell) [inline server/services/userAuthenticationService.js:16]: this fails on windows - no need to convert date to a string

File: server/views/autherror.ejs [hunk 3]
```diff
@@ -1,15 +1,15 @@
 <!DOCTYPE html>
 <html lang="en" class="govuk-template">
 <head>
-  <% include ./partials/head %>
+  <%- include ('./partials/head') -%>
```
Comment (steverendell) [inline server/views/autherror.ejs:4]: the new version of ejs no longer supports the syntax without brackets

File: server/app.js [hunk 4]
```diff
@@ -105,20 +104,6 @@ module.exports = function createApp({ signInService }) {
...
-        src: path.join(__dirname, '../assets/sass'),
-        dest: path.join(__dirname, '../assets/stylesheets'),
-        debug: true,
-        outputStyle: 'compressed',
-        prefix: '/stylesheets/',
-        includePaths: ['node_modules/govuk-frontend', 'node_modules/@ministryofjustice/frontend'],
-      }),
-    )
-  }
-
```
Comment (steverendell) [inline server/app.js:121]: you will need npm run build now

**Review by monskapinga — APPROVED** [review]

**Pull request merged**